### PR TITLE
Update Bootstrap to v. 5 and various UI fixes

### PR DESCRIPTION
**Pull request by bedo2991**
Graphical visualization of the changes, here: https://docs.google.com/document/d/1YbXeZ4Ug0Vi_P3J0pkjxSRvQVTgLdYchPLqOE-YDgw0/edit

### CODE REVIEW

File: resources/assets/ts/CreateEditEvent.ts [hunk 1]
```diff
@@ -69,13 +69,13 @@ $(() => {
             }
         }
     });
-  $('#templateSelector').selectpicker({
+  /*$('#templateSelector').selectpicker({
```
Comment (P1tt187) [inline resources/assets/ts/CreateEditEvent.ts:72]: please remove the dead code

File: resources/assets/ts/ShiftType.ts [hunk 2]
```diff
@@ -2,16 +2,16 @@
...
-  $('.shiftTypeSelector').selectpicker({
-    style: 'btn btn-secondary btn-sm',
-    liveSearch: true
-  });
-  $('.shiftTypeReplaceSelector').selectpicker({
-    style: 'btn btn-secondary btn-sm',
-    liveSearch: true
-  });
-});
+// $(window).on('load', () => {
```
Comment (P1tt187) [inline resources/assets/ts/ShiftType.ts:5]: remove dead code

File: resources/assets/ts/common/init.ts [hunk 3]
```diff
@@ -8,9 +8,9 @@ let checkInputSupport = (type: String) => {
 
 $(window).on({
   load: () => {
-    $('#userGroupDevelop').selectpicker({
-      style: 'btn-sm',
-    });
+    // $('#userGroupDevelop').selectpicker({
```
Comment (P1tt187) [inline resources/assets/ts/common/init.ts:11]: dead code

File: resources/assets/ts/vedst-scripts.ts [hunk 4]
```diff
@@ -176,10 +176,8 @@ $(function(){
 // Create/edit view //
 //////////////////////
 
-
-
 // Shows dynamic form fields for new job types
-$(document).ready(function() {
+jQuery(function() {
```
Comment (P1tt187) [inline resources/assets/ts/vedst-scripts.ts:180]:
```suggestion
jQuery(() => {
```

File: resources/assets/ts/vedst-scripts.ts [hunk 5]
```diff
@@ -193,17 +191,19 @@ $(document).ready(function() {
     };
 
     // Add one more job with every click on "+"
-    $('.btnAdd').click(function() {
-        let elementToCopy = $(this).closest('.box');
+    $('.btnAdd').on('click', function(e) {
```
Comment (P1tt187) [inline resources/assets/ts/vedst-scripts.ts:194]:
```suggestion
    $('.btnAdd').on('click', (e) => {
```

File: resources/views/partials/filter.blade.php [hunk 6]
```diff
@@ -1,53 +1,99 @@
...
-        </option>
-    </select>
+        <div class="form-check form-switch">
+            <input class="form-check-input palette-Purple-900-Primary bg me-3 ms-1" type="checkbox" role="switch" id="section-filter-survey" 
+            data-section-id="survey"
+            data-section="survey">
+            <label class="form-check-label badge palette-Purple-900-Primary bg align-text-bottom" for="section-filter-survey"> {{__('mainLang.survey')}}</label>
+        </div>
+
+        <!--
```
Comment (P1tt187) [inline resources/views/partials/filter.blade.php:48]: why is this block commented out?

File: resources/assets/ts/DarkMode.ts [hunk 7]
```diff
@@ -0,0 +1,80 @@
+/*!
```
Comment (P1tt187) [inline resources/assets/ts/DarkMode.ts:1]: where is this file from?
do we have to mention this in the readme?

**Review by P1tt187 — CHANGES_REQUESTED** [review]
looks good so far, i will try to test it tomorow
it looks that there are only minor changes needed

File: resources/assets/ts/DarkMode.ts (hunk 7 above)
Comment (bedo2991) [reply resources/assets/ts/DarkMode.ts:1]: From here: https://getbootstrap.com/docs/5.3/customize/color-modes/#javascript

**Comment by P1tt187** [comment]
found a bug in the statisctics view

steps to produce the problem:
1. open statistics view
2. activate a tab for a section
3. change the month

Same happens on the year statistic

Screenshot_20230801_233039

**Comment by P1tt187** [comment]
create an event from template is not working anymore
it shows a static list of entries

Screenshot_20230801_233500

**Comment by P1tt187** [comment]
open the template management results in `Attempt to read property "title" on null`

**Comment by P1tt187** [comment]
edit user is looking strange
Screenshot_20230801_234041

**Comment by P1tt187** [comment]
also assingning and removing permissions does not work anymore

**Comment by bedo2991** [comment]
> rom template is not working anymore
> it shows a static list of entries
Can't reproduce
image

**Comment by P1tt187** [comment]
> > rom template is not working anymore
> > it shows a static list of entries
> > Can't reproduce
> > image

looks like 
`php artisan migrate:fresh --seed` is not creating templates anymore?!

**Comment by bedo2991** [comment]
> looks like `php artisan migrate:fresh --seed` is not creating templates anymore?!

It's working for me:
```Seeding: ShiftsTableSeeder
Seeded:  ShiftsTableSeeder (130,813.79ms)
Seeding: TemplatesTableSeeder
Seeded:  TemplatesTableSeeder (14,455.46ms)
Seeding: SurveysTableSeeder
```
And I get data. I also can't reproduce the error with the template management page.

**Comment by P1tt187** [comment]
> > looks like `php artisan migrate:fresh --seed` is not creating templates anymore?!
> 
> It's working for me:
> 
> ```
> Seeded:  ShiftsTableSeeder (130,813.79ms)
> Seeding: TemplatesTableSeeder
> Seeded:  TemplatesTableSeeder (14,455.46ms)
> Seeding: SurveysTableSeeder
> ```
> 
> And I get data. I also can't reproduce the error with the template management page.

strange, looks like my local setup is broken

**Review by P1tt187 — APPROVED** [review]
so far looks good to me, i will deploy it on the demo page

**Pull request merged**